flake8_async/visitors/visitor_utility.py

```diff
@@ -117,11 +117,13 @@ def __init__(self, *args: Any, **kwargs: Any):
         # see imports
         if self.options.anyio:
             self.add_library("anyio")
+        if self.options.asyncio:
+            self.add_library("anyio")
```

Comment:
```suggestion
        if self.options.asyncio:
            self.add_library("asyncio")
```

Reply:
huh, surprised this was not caught by a linter for "different branches of if with same content should be combined with `or`"

Reply:
oh derp, it's because they're separate `if`s - not different branches for the same `if`.